src/Joint.h

```diff
@@ -244,6 +277,11 @@ class Joint
 	int dof_;
 
 	std::string name_;
+
+	bool is_mimic_ = false;
+	std::string mimic_name_ = "";
+	double mimic_multiplier_ = 1.0;
+	double mimic_offset_ = 0.0;
```

Comment:
All properties in RBDyn are camelCase ending with an underscore, maybe you had too much Python :snake: